tests/unit/static/golden/api.trulens.3.11.yaml

```diff
@@ -60,7 +60,7 @@ trulens.benchmark.test_cases:
     generate_summeval_groundedness_golden_set: builtins.function
 trulens.core:
   __class__: builtins.module
-  __version__: 1.1.0
+  __version__: 1.0.2a0
```

Comment:
bad merge?

Reply:
`1.0.2a0` seems to be accurate relative to what pyproject.toml states on main. What is it supposed to be?

Reply:
https://github.com/truera/trulens/pull/1539 probably needs to be merged in?

Reply:
Ahh - just merged it